fedimint-server/src/consensus/mod.rs

```diff
@@ -443,6 +453,69 @@ impl FedimintConsensus {
         epoch_history
     }
 
+    /// If the client config hash isn't already signed, aggregate signature shares from peers
+    /// dropping those that don't contribute.
+    async fn save_client_config_sig(
+        &self,
+        dbtx: &mut DatabaseTransaction<'_>,
+        outcome: &HbbftConsensusOutcome,
+        drop_peers: &mut Vec<PeerId>,
+    ) {
+        let config = self.get_config_with_sig(dbtx).await;
+
+        if config.client_hash_signature.is_none() {
+            let maybe_client_hash = config.client.consensus_hash(&self.module_inits);
+            let client_hash = maybe_client_hash.expect("hashes");
+            let peers: Vec<PeerId> = outcome.contributions.keys().cloned().collect();
```

Comment:
Var name could be more descriptive, I suspected a bug where we ban non-contributing peers at first.